tests/tool_excimer_script_metadata_test.php

```diff
@@ -26,7 +26,15 @@
  */
 class tool_excimer_script_metadata_test extends \advanced_testcase {
 
-    public function test_stripparamters(): void {
+    /**
+     * Set up before each test
+     */
+    protected function setUp(): void {
+        parent::setUp();
+        $this->resetAfterTest();
+    }
+
+    public function test_stripparamters() {
```

Comment:
Misspelling here. Also I think as previous comments mentioned, it would read better as `test_strip_parameters`